client/util/formatDate.test.ts

```diff
@@ -0,0 +1,85 @@
+import { formatDistance, formatToLocaleString, defaultEnvironment, Environment } from './formatDate'
+import { subDays } from 'date-fns'
+import { ja, enUS } from 'date-fns/locale'
+
+// memo(otofune): This requires that process.env.TZ equals to 'UTC'. (;;)
```

Comment:
Just a suggestion, how about overwriting `process.env.TZ` with this file to reduce the scope of impact?

Reply:
After node v8, we can't change TZ with changing `process.env.TZ`. see also: https://github.com/nodejs/node/issues/3449